test/fixtures/comparison-files/lib/viewers/ResultsViewer.test.ts/one-multilocation-violation-details.txt

```diff
@@ -5,6 +5,6 @@
     message:   This is a message
     locations:
         __PATH_TO_FILE_A__:20:1
-        (main) __PATH_TO_FILE_Z__:2:1 This is a comment at Location 2
-        __PATH_TO_FILE_A__:1:1 This is a comment at Location 3
-    resources: https://example.com/stub1RuleA
+        (main) __PATH_TO_FILE_Z__:2:1 "This is a comment at Location 2"
```

Comment:
Why is this just `2:1` instead of `2:1-7`?

Reply:
Good question. The way Core is currently implemented over at https://github.com/forcedotcom/code-analyzer-core/blob/dev/packages/code-analyzer-core/src/results.ts#L72 is to say that we do not actually return an endColumn if it doesn't have an associated endLine since it'll force clients to each make their own decisions as to what to do. It's just easier to that engines must supply endLine if they want to supply endColumn.

Reply:
But then why does this violation have an `endColumn` at all in the declaration? (see 149 of the test file).

Reply:
It doesn't - it's just a sanity check in case we ever change what the core module does... I want the test to fail somehow so we know what to do with it.